partner_contact_age_range/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Contact's Age Range",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Druidoo, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/partner-contact",
+    "category": "Customer Relationship Management",
+    "summary": "Age field for Contact's",
+    "depends": ["partner_contact_birthdate"],
+    "data": [
+        "security/ir.model.access.csv",
```

Comment:
I'd rather do an xml to help inheritance.

Reply:
@rousseldenis : we can also inherit this for eg.

access record in module:

`access_res_partner_age_range_user,access_res_partner_age_range_user,model_res_partner_age_range,base.group_user,1,0,0,0`

inherit it in other module:
`
partner_contact_age_range.access_res_partner_age_range_user,access_res_partner_age_range_user,model_res_partner_age_range,base.group_user,1,1,0,0`